src/bundling.ts

```diff
@@ -368,6 +368,13 @@ export class BundleExecutor {
           this._descriptor.requestDiscriminatorFields);
       return apiCall(request, callback);
     }
+    if (request[this._descriptor.bundledField] === undefined) {
+      console.warn(
```

Comment:
This is generally a no-no.  You want to use `process.emitWarning`:
https://medium.com/@jasnell/introducing-process-warnings-in-node-v6-3096700537ee

You kinda also want to be careful not to give a warning too many times.  You can see how I work with this stuff here:
https://github.com/googleapis/google-auth-library-nodejs/blob/master/src/messages.ts

Reply:
OK! I actually copied this `warn` stuff from the above line :) I will refactor that as well then.

Reply:
Done, PTAL!